webapp/frontend/app/routes/lambda-instances/index.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
```

Comment:
Please, add AuthenticatedMixin to this route as done in [1].

[1] https://github.com/grnet/okeanos-LoD/blob/devel/webapp/frontend/app/routes/create-lambda-instance.js